doc/docs/Python_User_Interface.md

```diff
@@ -7022,26 +7007,13 @@ value will persist across runs within the same script.
 
 </div>
 
-
-<a id="verbosity"></a>
-
-```python
-def verbosity(v=1): 
-```
-
-<div class="function_docstring" markdown="1">
-
-Given a number `v`, specify the degree of Meep's output: `0` is quiet mode, `1` (the
-default) is ordinary output, `2` is extra debugging output, and `3` is all debugging
-output.
-
-</div>
+@@ verbosity @@
```

Comment:
Seems the id tag for `verbosity` is not being recognized or is misplaced in `Python_User_Interace.md`?

Reply:
It's because it's not a function any more. I added tags for the `Verbosity` class in my other PR, and added text to the docstring to mention that there is an instance of the class named `verbosity`